pkgs/misc/arm-trusted-firmware/default.nix

```diff
@@ -151,6 +151,22 @@ in {
     platformCanUseHDCPBlob = true;
   };
 
+  armTrustedFirmwareRK3588 = buildArmTrustedFirmware rec {
+    extraMakeFlags = [ "bl31" ];
+    platform = "rk3588";
+    extraMeta.platforms = ["aarch64-linux"];
+    filesToInstall = [ "build/${platform}/release/bl31/bl31.elf"];
+    platformCanUseHDCPBlob = true;
+    # TODO: remove this condition once mainline ATF has support for the rk3588 platform
```

Comment:
Is there any upstream CL that we could link to in the comment?

Reply:
Arm trusted firmware uses Gerrit, that's their unit of code review, compared to PRs on GitHub.

Reply:
I'm aware of that, though I still don't know what the term "CL" is

Reply:
It means "changelist". See https://commondatastorage.googleapis.com/chrome-infra-docs/flat/depot_tools/docs/html/git-cl.html for a reference to it.